modules/atlas_account/default/0.1/facets.yaml

```
@@ -2,11 +2,46 @@ intent: atlas_account
 version: '0.1'
 description: Adds atlas_account - default flavor
+outputs:
+  default:
+    type: "@outputs/atlas_account"
+    providers:
+      mongodbatlas:
+        source: mongodb/mongodbatlas
+        version: 1.17.0
+        attributes:
+          public_key: attributes.public_key
+          private_key: attributes.private_key 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)